ros/src/actuation/vehicles/packages/as/nodes/ssc_interface/ssc_interface.cpp

```diff
@@ -19,7 +19,7 @@
 SSCInterface::SSCInterface() : nh_(), private_nh_("~"), engage_(false), command_initialized_(false)
 {
   // setup parameters
-  private_nh_.param<bool>("use_rear_wheel_speed", use_rear_wheel_speed_, true);
+  // private_nh_.param<bool>("use_rear_wheel_speed", use_rear_wheel_speed_, true); #Redudant, being estimated in SSC
```

Comment:
Please remove these items instead of commenting them out. We can always roll back in the commit history if we need to see what was here.